recipes-devtools/cargo/cargo_1.61.0.bbappend

```diff
@@ -1,3 +0,0 @@
-unset OPENSSL_SYS_USE_PKG_CONFIG
```

Comment:
we need openssl 3.1 support for Kirkstone. Any thoughts on that?

Reply:
OpenSSL 3 is being used

I'm using Rust+Cargo 1.64/1.66.  This bbappend does not apply to our BSP.

Reply:
ok. I understand this is preliminary one and already has temp sensor working. Is there any more work needed to support it?

Reply:
No, I think it is working (at least for the cases I've tested)